test/karma.conf.js

```diff
@@ -18,7 +18,10 @@ module.exports = function (config) {
         },
         browserify: {
             debug: true,
-            transform: [['babelify', { 'presets': ['es2015'] }]]
+            transform: [['babelify', {
+                "presets": ["@babel/preset-env"],
```

Comment:
if preset-env without specifying any targeted browser is equivalent to es2015 language level, than I'm fine with this change.

Reply:
transform synced with our other components babel transform setup